http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpDockerDAOSpec.scala

```diff
@@ -175,7 +175,7 @@ class HttpDockerDAOSpec extends AnyFlatSpec with Matchers with BeforeAndAfterAll
         |}
         |""".stripMargin
     val expectedResult =
-      DecodingFailure("Missing required field", List(DownField("Env"), DownField("container_config")))
+      DecodingFailure("Missing required field", List(DownField("Env")))
```

Comment:
This change gets tests passing. But I'm not sure that this is the desired behavior - it seems like the message is expected to show the object heirarchy of the failing node (container_config > Env) and instead it's just showing the latest. Since it's just a message presumably it's NBD but wanted to flag it for someone who might understand this code a little better